ansible_collections/arista/avd/plugins/plugin_utils/utils/template.py

```diff
@@ -33,6 +36,16 @@ def template(template_file, template_vars, templar, searchpath):
     j2template, dummy = loader._get_file_contents(template_file_path)
     j2template = to_text(j2template)
 
-    templar.available_variables = template_vars
-    result = templar.template(j2template, convert_data=False, escape_backslashes=False)
+    cache_dir = Path(PurePath(template_file_path).parent, ".j2cache")
```

Comment:
We can do it this way instead of creating the Path twice

```suggestion
    cache_dir = Path(template_file_path).parent.joinpath(".j2cache")
```

Reply:
implemented in https://github.com/aristanetworks/ansible-avd/pull/2307/commits/34f55e3e7773eb18da19bba7acb4af280a8422ba